vault/resource_okta_auth_backend.go

```diff
@@ -444,6 +458,19 @@ func oktaAuthBackendUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("tune") {
+		if raw, ok := d.GetOk("tune"); ok {
+			log.Printf("[INFO] Writing okta auth tune to '%q'", oktaTuneEndpoint(path))
+
+			err := authMountTune(client, oktaTuneEndpoint(path), raw)
+			if err != nil {
+				return nil
+			}
+
+			d.SetPartial("tune")
```

Comment:
The code won't build as-is. [SetPartial has been removed](https://developer.hashicorp.com/terraform/plugin/sdkv2/guides/v2-upgrade-guide#removal-of-helper-schema-resourcedata-setpartial) in the terraform sdk v2.